src/calculateRentalCost.js

```diff
@@ -20,7 +20,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
+  }
+  if (days >= 3 && days < 7) {
```

Comment:
no need for this check
```suggestion
  if (days < 7) {
```